shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -89,6 +90,11 @@ FlutterDesktopGetPluginRegistrar(FlutterDesktopEngineRef engine,
 FLUTTER_EXPORT FlutterDesktopMessengerRef
 FlutterDesktopEngineGetMessenger(FlutterDesktopEngineRef engine);
 
+// Posts an app control to the engine instance.
+FLUTTER_EXPORT void FlutterDesktopNotifyAppControl(
+    FlutterDesktopEngineRef engine,
+    app_control_h app_control);
```

Comment:
This breaks the x64 build.

I don't think it makes sense to use a switch like `#ifndef __X64_SHELL__` inside the public embedder API. Can we use `void*` (or typedef to it) as an argument type and reinterpret_cast it before accessing?